src/main/java/frc/robot/subsystems/FourBar.java

```diff
@@ -61,8 +63,8 @@ public FourBar() {
 
     @Override
     public void periodic() {
-        // This method will be called once per scheduler run
-
+        fourBarfollower.set(ControlMode.PercentOutput, 0);
```

Comment:
Temporary change because the motors were uncapable to lift the four bar, even at 100% output. Because of this, I set them to 0% output, so that we would neither damage the motors, nor get that annoying noise